packages/nextjs-template/components/DendronSearch.tsx

```diff
@@ -42,10 +48,13 @@ type SearchFunction = (
   setResults: (results: SearchResults) => void
 ) => void;
 
-function DebouncedDendronSearchComponent(props: DendronPageWithNoteDataProps) {
+function DebouncedDendronSearchComponent(props: {
+  notes: SectionsData["notes"];
+  indexId: string;
+}) {
   // Splitting this part from DendronSearchComponent so that the debounced
   // function doesn't get reset every time value changes.
-  const results = useFetchFuse(props.notes);
+  const results = useFetchFuse(props.notes as NotePropsDict);
```

Comment:
don't cast `props.notes` as something its not. change type signature instead or use a new method